content/blog/2022-06-28-serving-models-with-mlem.md

```diff
@@ -0,0 +1,277 @@
+---
+title: Serving Models with MLEM
+date: 2022-06-28
+description: >
+  Once you have a model that's ready for production, getting it out can be a
+  complicated step. In this tutorial, we're going to use MLEM to deploy a model
+  to an API.
+descriptionLong: >
+  Getting models out to our end users is usually a difficult task. You have to
+  make sure it's in the correct format for the environment and for the way it
+  will be consumed. We're going to walk through how you can use MLEM to save
+  your model and serve it through an API.
+picture: 2022-06-28/mlem-deploys.png
+pictureComment: Serving Models with MLEM
+author: milecia_mcgregor
+commentsUrl: https://discuss.dvc.org/t/syncing-data-to-aws-s3/1192
+tags:
+  - MLOps
+  - DVC
+  - Git
+  - MLEM
+  - Collaboration
+---
+
+Training a machine learning model is only one step in the process of getting
+something useful out to the end users. The goal of machine learning projects to
+make an accurate model that provides predictions based on the data received from
+a production environment. That could mean making predictions on data coming from
+an API or do some batch predictions.
+
+Either way, you'll need to save your trained and validated model in format
+that's consumable by other systems. That's why we'll be covering how to serve
+models through an endpoint with [MLEM](https://mlem.ai/). We'll start by
+training a simple model. You can get the repo we're working with
+[here](https://github.com/iterative/stale-model-example/tree/mlem-serve).
+
+## Train a model
+
+There are instructions in the project
+[README](https://github.com/iterative/stale-model-example/tree/mlem-serve#readme)
+on how to get everything you need installed and running. Make sure you run a few
```

Comment:
I think we should explain that it's a DVC project and that MLEM and DVC play well together.